pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -326,6 +354,18 @@ func (mi *StorageInitializerInjector) InjectStorageInitializer(pod *v1.Pod) erro
 		}
 	}
 
+	// Update container spec from a storage container CR if there is a match
+	storageContainerSpec, err := getContainerSpecForStorageUri(srcURI, mi.client)
+	if err != nil {
+		return err
+	}
+	if storageContainerSpec != nil {
```

Comment:
What happens if `storageContainerSpec` is nil and nothing is matched?

Reply:
I'm thinking we can use config map for now, which can be removed in future versions. I added a comment here.